integration_tests/e2e/refer.cy.ts

```diff
@@ -352,6 +353,128 @@ context('Refer', () => {
     })
   })
 
+  describe('When adding programme history', () => {
```

Comment:
Can this be moved back to the lower section alongside the other programme history stuff?